framework/aster-frame/src/task/processor.rs

```diff
@@ -38,44 +38,42 @@ impl Processor {
     }
 }
 
-lazy_static! {
-    static ref PROCESSOR: Mutex<Processor> = Mutex::new(Processor::new());
+cpu_local! {
+    static PROCESSOR: SpinLock<Processor> = SpinLock::new(Processor::new());
```

Comment:
Is `RefCell` a more suitable choice here than `SpinLock`? Considering that `PROCESSOR` is CPU-local, it won't be accessed by multiple threads simultaneously, correct?